tests/refarm_pagination/test_pagination.py

```diff
@@ -1 +1,67 @@
-# @todo #556:30m Create unit tests for pagination classes.
+from django.conf import settings
+from django.core import paginator
+from django.test import TestCase
+from django.http import Http404
+
+from refarm_pagination import pagination
+
+
+class Paginator(TestCase):
+
+    def test_page_404error(self):
+        with self.assertRaises(Http404):
+            pagination.Paginator([], 1).page(2)
+
+    def test_page_result(self):
+        self.assertIsInstance(
+            pagination.Paginator([], 1).page(1),
+            paginator.Page,
+        )
+
+
+class NeighborPage(TestCase):
+
+    def test_first_page_url(self):
+        """First page gives url without `page` query string param."""
+        base_url = 'url'
+        self.assertEqual(
+            base_url,
+            pagination.NeighborPage(1).url(base_url),
+        )
+
+    def test_rest_page_urls(self):
+        """Any page gives url with `page` query string param exept the first page."""
+        page_number = 2
+        base_url = 'url'
+        self.assertEqual(
+            f'{base_url}?page={page_number}',
+            pagination.NeighborPage(page_number).url(base_url),
+        )
+
+
+class NeighborPages(TestCase):
+
+    def neighbors(self, object_count, per_page, page_number):
+        return pagination.NeighborPages(
+            pagination.Paginator(
+                list(range(object_count)),
+                per_page,
+            ).page(page_number),
+        )
+
+    def test_neighbors(self):
```

Comment:
it's too common name for tested case.
`test_current_page_is_not_neighbour` or `test_neighbours_list_data` is more good, i think.
But decide